src/frontend/src/components/importer/ImporterDrawer.tsx

```diff
@@ -81,6 +80,7 @@ export default function ImporterDrawer({
         return 3;
       case importSessionStatus.COMPLETE:
         return 4;
+      default:
```

Comment:
The 'default' case should return a value here still, otherwise you are changing functionality.

Reply:
Are you sure? `default` is empty in the current code base